custom-idl/IndexedDB.idl

```diff
@@ -1,8 +1,3 @@
-partial interface IDBDatabase {
-  // https://github.com/mozilla/gecko-dev/blob/0dfbe5a699cc6c73cf8c14d1aa10ba10ef3ec8fa/dom/webidl/IDBDatabase.webidl#L43
-  IDBRequest createMutableFile(DOMString name, optional DOMString type);
```

Comment:
It's because this is a feature that isn't in BCD already, and is Gecko-only.  I'm not finding this method in the spec, so I figure we should remove it and prevent its generation in `add-new-bcd`.

Reply:
Ah, so I would like to keep anything that is in any browser, and at a later point go through it all and deciding what to add to BCD and not.

Would it be easy to trim this to just stuff that isn't implemented anywhere currently? I guess not?

Reply:
Okay, so we should probably mark this PR as a draft, in that case.  This PR contains all the results that are added in `add-new-bcd`, which excludes everything that's all `false`.

Reply:
Oh, so this PR actually doesn't delete anything that's in the category I was hoping for?

Reply:
Unfortunately it doesn't, no -- everything removed in this PR is added by `add-new-bcd`, meaning that they were all supported in some browser at some version.

Reply:
I see. Is there a simple hack to list the features that aren't in BCD and that the collector would populate as all false? Those are the ones I'd like to remove :)

Reply:
What we could do is run `add-new-bcd` with all of these still in the data, then check the results of `find-missing` and remove those!  I'll start working on a PR to do just that!

Reply:
Ah, clever approach!

Reply:
Gave it some more thought, and for now I'm going to hold off on writing the desired PR.  Since there's no harm in their results being included, it won't affect our current goals.